pandas/tests/scalar/timestamp/test_timestamp.py

```diff
@@ -1092,3 +1094,23 @@ def test_constructor_ambigous_dst():
     expected = ts.value
     result = Timestamp(ts).value
     assert result == expected
+
+
+@pytest.mark.xfail(
+    LooseVersion(get_distribution("python-dateutil").version) < LooseVersion("2.7.0"),
```

Comment:
@jreback 
I apologize for `get_distribution("python-dateutil").version`. It's pretty horrible.
Unfortunately `from dateutil import __version__` breaks mypy checks. And `dateutil.__version__` breaks it.

I don't know why, because mypy is completely fine with `from pandas import __version__`. Both packages have the attribute and they are `str`. I checked (this check fail has been pretty hard on my nerves). 

Tried other stuff too. Only dateutil fails:
```
$ mypy pandas/tests/scalar/timestamp/test_timestamp.py
pandas\tests\scalar\timestamp\test_timestamp.py:1: error: Module 'dateutil' has no attribute '__version__'
Found 1 error in 1 file (checked 1 source file)
(pandas-dev)
```

My current solution passes all the checks, but I would appreciate suggestions if anyone sees how the ugliness can be avoided without breaking mypy.